pkg/config/validation/validation_test.go

```diff
@@ -4984,6 +4982,32 @@ func TestValidateServiceEntries(t *testing.T) {
 			valid:   true,
 			warning: true,
 		},
+		{
+			name: "dns round robin with more than one endpoint", in: &networking.ServiceEntry{
+				Hosts:     []string{"google.com"},
+				Addresses: []string{},
+				Ports: []*networking.Port{
+					{Number: 8081, Protocol: "http", Name: "http-valid1"},
+				},
+				Endpoints: []*networking.WorkloadEntry{
+					{
+						Address: "1.1.1.1",
```

Comment:
are IP address for DNS invalid anyways?

Reply:
they are not, It is not being validated because the top level vaildation is kicking in. I will fix it though